### PR TITLE
Prepare 1.x draft version; restore discussion of W3C VC credential source

### PR DESCRIPTION
**Pull request by scouten-adobe**

### CODE REVIEW

**Review by OR13 — CHANGES_REQUESTED** [review]
I would like to see this target W3C VC Data Model v2

I don't think it is good to try to target both v1 and v2

File: docs/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -110,6 +110,8 @@ Eve is a musician with a talent for releasing songs featuring clever lyrics and
 == Normative references
 
 * https://c2pa.org/specifications/specifications/2.0/specs/C2PA_Specification.html[C2PA technical specification, version 2.0]
+* https://www.w3.org/TR/vc-data-model/[W3C verifiable credentials data model, version 1.1]
```
Comment (OR13) [inline docs/modules/ROOT/pages/index.adoc:113]:
```suggestion
* https://www.w3.org/TR/vc-data-model-2.0/[Verifiable Credentials Data Model v2.0]
```

File: docs/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -440,7 +442,132 @@ The signature type is represented by the `sig_type` field and must be one of the
...
+In addition, the _<<W3C decentralized identifier document>>_ MUST contain an `assertionMethod` field as defined by link:++https://www.w3.org/TR/did-core/#assertion__[Section 5.3.2, “Assertion,”] of the W3C decentralized identifiers specification.
+
+An example of a compliant _<<W3C decentralized identifier document>>_ is shown in the example below, which is non-normative:
+
+[source,json]
+----
+{
+  "@context": [
+    "https://www.w3.org/ns/did/v1",
+    "https://w3id.org/security/suites/jws-2020/v1"
```
Comment (OR13) [inline docs/modules/ROOT/pages/index.adoc:516]:
```suggestion
    "https://www.w3.org/ns/security/jwk/v1"
```

File: docs/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -440,7 +442,132 @@ The signature type is represented by the `sig_type` field and must be one of the
...
+----
+
+[#vc-verification-methods]
+NOTE: TO DISCUSS (link:https://github.com/creator-assertions/identity-assertion/issues/18[issue #18]): Do we wish to have any further restrictions on verification methods beyond those specified in link:++https://www.w3.org/TR/did-core/#verification-material++[Section 5.2.1, “Verification material,”] of the DID spec? See also link:++https://www.w3.org/TR/did-spec-registries/#verification-method-types++[Section 6.1, “Verification method types,”] of the W3C DID specification registries document for a list of possible verification methods. Currently about 10 are listed.
+
+==== Verifiable credential requirements
+
+[#vc-structure]
+NOTE: TO DO (link:https://github.com/creator-assertions/identity-assertion/issues/19[issue #19]): Specify structure, including how list of _<<_referenced_assertions,referenced assertions>>_ is expressed in the verifiable credential.
+
```
Comment (OR13) [inline docs/modules/ROOT/pages/index.adoc:544]:
```suggestion

TODO: Add a table of supported media types for *secured* verifiable credentials. For example:

- application/vc+ld+json+cose
- application/vc+ld+json+jwt
- application/vc+ld+json+sd-jwt

TODO: Define which are mandatory to implement and which are optional.

```

**Comment by scouten-adobe** [comment]
@or13 please re-review when you have a moment. I've removed all references to VC 1.1

**Comment by scouten-adobe** [comment]
Closed in favor of #85 per discussion in 01 April 2024 meeting.

**Pull request closed without merging**